net-misc/youtube-viewer/youtube-viewer-3.2.4.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils gnome2-utils perl-module vcs-snapshot
+
+DESCRIPTION="A command line utility for viewing youtube-videos in Mplayer"
+HOMEPAGE="https://trizen.googlecode.com"
+SRC_URI="https://github.com/trizen/youtube-viewer/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
The correct url does not use `v`. The following works for me without the use of vcs-snapshot:

```
 net-misc/youtube-viewer/Manifest                    | 2 +-
 net-misc/youtube-viewer/youtube-viewer-3.2.4.ebuild | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/net-misc/youtube-viewer/Manifest b/net-misc/youtube-viewer/Manifest
index cb426ee..9b4eaf7 100644
--- a/net-misc/youtube-viewer/Manifest
+++ b/net-misc/youtube-viewer/Manifest
@@ -1,3 +1,3 @@
 DIST youtube-viewer-3.1.9.tar.gz 222276 SHA256 9b7ba2b50cea054898024bc871e231673269ab50db0526ee3f1a7ae62a51e8d2 SHA512 60ff81defd4d8d1bc1f6b5bceaa03c9bbe64e8fea2083ce60b04b6c14d8b13ded6da255b9aa46bfcb1b76cf33121ddb738d0f0c159e4bd207999ca558a43e86e WHIRLPOOL 2e6a79992c7760d93c2e28f1e4f5c14861b6fefed8e0bebb9a7d88c6cf27000904971b12274a972ca66471ffee7c3bfee8c90218c16ddd0ec96cc8f46779bffd
 DIST youtube-viewer-3.2.0.tar.gz 222528 SHA256 60a4ded8798da3cdadadf816b1c4e87c6feb697f01db70630f014584e5a2392e SHA512 bda3592874ded71e6a60d87b669d2c80999425ef2f2061e942a8db5755d9d6f2dabf038b8b7984efb1cccc3a5f9f12b2fa56290be9db871b2784b2c464367c3b WHIRLPOOL 73ec965bf4cf65d545b99d73e003ce7d74d92dcdc1f5d29c8b925e4d96e93d5924b66e06ec40e22f16119b6df6445344d650d5ca086e52aeb60a350c7ca8ce0d
-DIST youtube-viewer-3.2.4.tar.gz 224308 SHA256 e97e9386e23194a6e7161fb3f44f20e63f02c710b7dad019acdb9d102da43b43 SHA512 cfd2eb2f928ca26265a9d4c9e348935e1d43dd5b01298a90f69b6f26a93ebbdc7b782f6aa06111bb689540a620dc416d8b2ac3f8bf4f4eccaf540c0f3204bd4f WHIRLPOOL 28b1c0c105b83cfc0409eacf7d74315b718398d16ba18769a51f5ad2d794a95b27129b8ddb6ec126daa6774a6ec417817e6355b1319a93d77970ddf5100a855a
+DIST youtube-viewer-3.2.4.tar.gz 224282 SHA256 ca37024ce018e31bd4c99f5df208e842f44979c07c5d31647c1fbe4e17b1b296 SHA512 06bb763460b90f26aa1671c29056374df62d4e840acb3a800f4840160535fef30e60e6655b1d4ba7423a0424384acdfc00aa068a420aa836110974a5f7152ec2 WHIRLPOOL bfeb12bacd98b6da8ecb0ee4b626a8fdc98331f21049bdd99524d65ff5de109abe859c882dac5347fa07cee704d229e64496a369ae1c3ae6d24557c4ff142b59
diff --git a/net-misc/youtube-viewer/youtube-viewer-3.2.4.ebuild b/net-misc/youtube-viewer/youtube-viewer-3.2.4.ebuild
index 6281204..3d37b50 100644
--- a/net-misc/youtube-viewer/youtube-viewer-3.2.4.ebuild
+++ b/net-misc/youtube-viewer/youtube-viewer-3.2.4.ebuild
@@ -4,11 +4,11 @@
 
 EAPI=6
 
-inherit eutils gnome2-utils perl-module vcs-snapshot
+inherit eutils gnome2-utils perl-module
 
 DESCRIPTION="A command line utility for viewing youtube-videos in Mplayer"
 HOMEPAGE="https://trizen.googlecode.com"
-SRC_URI="https://github.com/trizen/youtube-viewer/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/trizen/youtube-viewer/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
```